week-1/schedule.md

```diff
@@ -0,0 +1,68 @@
+| 09:45   | Intro reading                                                                                                                                                      | Accessibility                     |
+| 10:00   | [Design burst: grids](https://docs.google.com/presentation/d/11rIDhbzacs6AUFTVCiJd9f0Ud73TvM4Q9LScMVhctUY/edit?ts=5b4f0840#slide=id.g26a95a14fb_0_0)               |                                   |
+| 10:30   | Accessibility workshop                                                                                                                                             | Accessibility, pair programming   |
+| 11:30   | [Git workshop](https://github.com/foundersandcoders/git-workflow-workshop-for-two)                                                                                 | Version control, pair programming |
+| _13:00_ | _Lunch_                                                                                                                                                            |                                   |
+| 14:00   | [Git workshop contd.](https://github.com/foundersandcoders/git-workflow-workshop-for-two)                                                                          |                                   |
+| 14.30   | Project intro                                                                                                                                                      |                                   |
+| 14:45   | [Technical spikes and presentations intro](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/general/research-presentation-guidance.md) |                                   |
+| 15:00   | [Technical spikes](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/week-1/research-afternoon.md)                                      |                                   |
+| 16:30   | Spike presentation prep                                                                                                                                            |                                   |
+| 17:00   | Spike presentations                                                                                                                                                |                                   |
+
+## Day 3
+
+| Time    | Activity                                                                     | Learning outcomes |
+| ------- | ---------------------------------------------------------------------------- | ----------------- |
+| 09:45   | Intro reading                                                                | CSS layouts       |
+| 10:00   | [CSS Gallery MC](https://github.com/foundersandcoders/css-gallery-challenge) | CSS Media queries |
```

Comment:
Happy to leave this there for now, but I'm not sure it's a great intro to media queries. I think an image gallery is better suited to the CSS grid design burst stuff .